packages/client/src/pages/Trecipe/TrecipePage.tsx

```diff
@@ -202,13 +227,13 @@ class TrecipePage extends React.Component<TrecipeProps, TrecipeState> {
                     (dest) => dest.destUUID === destination.uuid && !dest.completed
                 )
             ) {
-                if (this.props.user.username) {
+                if (this.props.user.user.username) {
```

Comment:
Minor: Can we use something other than `this.props.user.user` lol. Maybe we can split up user and isAuthenticated in mapStateToProps.

Reply:
lol for sure, I will split it in mapStateToProps